apps/p42_log_plugin_es/mix.exs

```diff
@@ -19,13 +19,14 @@ defmodule Proxy42.Plugins.Logging.Elasticsearch.Mixfile do
     [ {:poolboy, "~> 1.5"},
       {:hackney, "~> 1.13"},
       {:jsx, "~> 2.9"},
+      {:datum, "~> 4.5"},
       {:proxy42, in_umbrella: true},
     ]
   end
 
   defp env() do
     [ 
-      es_url: "http://localhost:9200/proxy42/logs/_bulk",
+      es_url: "http://localhost:9200/p42logs/_bulk",
```

Comment:
This breaks compatibility with ES v6 and below.

Reply:
Actually, concept of "document type" is dead in Elastic.

https://www.elastic.co/guide/en/elasticsearch/reference/current/removal-of-types.html 

> The preferred type name is _doc, so that index APIs have the same path as they will have in 7.0

I would not look on any Elastic version below 6.x

All-in-all, the integration testing is still required before making a final conclusion.

Reply:
The stable version is currently v7. Vast majority of systems in the wild are not v7. The mapping types are discontinued only from v8 and upwards, v7 still works with deprecation warnings.

It's mostly a matter of preference at this point I guess. Since it is just a config change, I won't contest this much. Let's just add a note to docs mentioning this. I'm writing up docs, will add the note there.

Reply:
ok